lms/djangoapps/learner_dashboard/views.py

```diff
@@ -73,4 +74,7 @@ def program_details(request, program_uuid):
         'user_preferences': get_user_preferences(request.user)
     }
 
-    return render_to_response('learner_dashboard/program_details.html', context)
+    if waffle.switch_is_active('new_program_progress'):
+        return render_to_response('learner_dashboard/new_program_details.html', context)
```

Comment:
Ok I can change all the new to 2017. I wanted to give a name that we wouldn't forget to change in the future, but 2017 seems like it would also work.  
I'd probably rather not use v2 because I could imagine us not deleting that part of the name